### PR TITLE
Rename generic rpm installers

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Addresses https://github.com/aspnet/Universe/issues/866

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
@natemcmaster Is there anything in dotnet/cli that we need to update to account for the naming change?

**Comment by natemcmaster** [comment]
yes. Here: https://github.com/dotnet/cli/blob/release/2.1.3xx/build/BundledRuntimes.props#L36

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Microsoft.AspNetCore              | [![][metapackage-myget-badge]][metapackage-m
 [linux-x64-tar]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-latest-linux-x64.tar.gz
 [osx-x64-tar]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-latest-osx-x64.tar.gz
 [debian-x64-deb]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-latest-x64.deb
-[redhat-x64-rpm]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-latest-rhel.7-x64.rpm
+[redhat-x64-rpm]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-latest-x64.rpm
```
Comment (natemcmaster) [inline README.md]: Just noticing now that the README only contains links to dev builds, not release/2.1. Maybe we should backport https://github.com/aspnet/Universe/pull/83 as well

Comment (natemcmaster) [reply README.md]: Wrong link: https://github.com/aspnet/Universe/pull/830

**Pull request merged**